src/robusta/utils/parsing.py

```diff
@@ -13,3 +14,10 @@ def load_json(s: Union[str, bytes]) -> Any:
         sep = "----------------"
         logging.error(f"could not parse json:\n{sep}\n{s}\n{sep}\n")
         raise
+
+
+def to_datetime_str(datetime_obj: datetime.datetime) -> str:
```

Comment:
I would give this a slightly more descriptive name like `datetime_to_db_str`. The comment is very good, but I would make the name a little better.

Reply:
fixing